src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul,
+li {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.block {
+  background-color: #ffff;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  margin-bottom: -2px;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-left: 2px;
+  margin-right: 2px;
+  margin-top: 2px;
+}
+
+.stars--n:nth-child(n) {
```

Comment:
_stars--n_ should not exist.
What you should do is modify the _.stars__star_ style based only in the modifier of the parent.
Here is an example:
```
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
... {
    background-image: url(images/star-active.svg);
}
```